tests/TestCase/Command/ModelCommandAssociationDetectionTest.php

```diff
@@ -160,6 +160,7 @@ public function testBakeAssociationDetectionProductVersionsTable()
     {
         $driver = ConnectionManager::get('test')->getDriver();
         $this->skipIf($driver instanceof Mysql, 'Incompatible with mysql');
+        $this->skipIf($driver instanceof Sqlite, 'Incompatible with sqlite');
```

Comment:
Why is this now incompatible with sqlite?

Reply:
As my comment states above: 
The only other custom change I had to do was the fact, that SQLite now seems to generate a `->nonNegativeInteger()` as a FK validator. See d18fea1
Thats why that tests needs to be skipped for the unsigned version of the test but done for the signed one